qemu/tests/cfg/dump_guest_core.cfg

```diff
@@ -13,6 +14,8 @@
     dump_guest_memory_file = "/usr/share/qemu-kvm/dump-guest-memory.py"
     x86_64:
         extra_params += " -device vmcoreinfo"
+    ppc64:
```

Comment:
Would you please share why ppc64 BE does not need this step?

Reply:
The gdb also needs kernel's packages so I'm afaid that it won't work as expected since the host's kernel is *.ppc64le.